clients/web/src/views/body/UsersView.js

```diff
@@ -60,14 +59,27 @@ var UsersView = View.extend({
             parentView: this
         }).on('g:resultClicked', this._gotoUser, this);
 
+        if (getCurrentUser() && getCurrentUser().get('admin')) {
+            var userCountpromise = UserCollection.getUsersCount()
+                .done((resp) => {
```

Comment:
I'd name this parameter `count`, rather than `resp`, since it's just an integer count, not a response body.